authenticator.go

```diff
@@ -104,7 +108,13 @@ func newTokenReviewAuthenticatorWithOpts(opts *AuthenticatorOptions) (authentica
 }
 
 // GetUsernameHeaderFromEnv retrieves from environment variable the name of the header
-// to use when authenticating requests by username header
+// to use when authenticating requests by username header to extract user's username
 func GetUsernameHeaderFromEnv() string {
 	return os.Getenv(EnvUsernameHeader)
 }
+
+// GetGroupsHeaderFromEnv retrieves from environment variable the name of the header
+// to use when authenticating requests by username header to extract user's groups
+func GetGroupsHeaderFromEnv() string {
+	return os.Getenv(EnvGroupsHeader)
+}
```

Comment:
We should probably test group membership in e2e tests somehow, maybe against something simple like `system:authenticated` or similar.  I'd like to see what changes we need to make to the nginx configuration to pass on group membership.

Reply:
That's right. I'm working at them in a separate PR